.github/workflows/node-integration.yml

```
@@ -2,7 +2,13 @@
 
 name: Node integration tests
 
-on: workflow_dispatch
+# Run this workflow always 
+on:
+  push:
+    branches: [ * ]
+  pull_request:
+    branches: [ * ]
```

I believe you can simplify this to:

```suggestion
on: [push, pull_request]
```